cfgov/jinja2/v1/_includes/organisms/content-banner.html

```diff
@@ -0,0 +1,33 @@
+{# ==========================================================================
+
+   Banner
+
+   ==========================================================================
+
+   Description:
+
+   Create an banner when given:
+
+    value:         				An object used to customize the markup.
+
+   	value.border_top:       	Whether banner should have a top border.
+
+   	value.border_bottom:    	Whether banner should have a bottom border.
+
+   	value.content:          	Banner content
+
+   	value.additional_classes: 	Additional banner classes
+
+   ========================================================================== #}
+	<div class="content_banner 
```

Comment:
Looks like this has been resolved, but for what it's worth, the header is an organism and the hero is a molecule, so this banner seems like it could be an atomic component, probably a molecule, but I'll leave that to you.